dell/precision/5520/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, config, ... }:
+
+{
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/gpu/nvidia/maxwell
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/cpu/intel/kaby-lake
+  ];
+
+  boot = {
+    kernelParams = [
+      # fix lspci hanging with nouveau
+      # source https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1803179/comments/149
+      "acpi_rev_override=1"
+      "acpi_osi=Linux"
+      "nouveau.modeset=0"
+      "pcie_aspm=force"
+      "drm.vblankoffdelay=1"
+      "nouveau.runpm=0"
+      "mem_sleep_default=deep"
+      # fix flicker
+      # source https://wiki.archlinux.org/index.php/Intel_graphics#Screen_flickering
+      "i915.enable_psr=0"
+      "nvidia_drm.modeset=1"
+    ];
+  };
+
+  hardware = {
+    nvidia = {
+      open = lib.mkDefault false;
```

Comment:
Afaik we do set this in nvidia generation specific profiles i.e. `../../../common/gpu/nvidia/maxwell`